python/lambdas/pause-preservica-activity/README.md

```diff
@@ -0,0 +1,64 @@
+# DR2 Pause Ingest
+
+## Input
+
+The lambda can be triggered either directly from a call to invoke-lambda in GitHub actions or can be triggered by an
+Eventbridge scheduled event.
+
+The invoke-lambda input will be either
+
+```json
+{
+  "pause": true
+}
+```
+
+or
+
+```json
+{
+  "pause": false
+}
+```
+
+depending on whether we want to pause and resume the ingest.
+
+With the scheduled event input, we're only looking at the `source` field
+
+```json
+{
+  "source": "aws.events"
+}
+```
+
+## Output
+
+The lambda doesn't return anything
+
+## Steps
+
+If `pause` is True
+
+* Send a message to Slack to say that Preservica activity is being paused
+* Disable the rotation for the all secrets.
```

Comment:
1. Should this be "Disable the rotation for all secrets" or "Disable the rotation for all of the secrets"
2. There doesn't need to be a full stop here